File: front/PortfolioDev/src/models/portfolio/portfolio.ts
import {Projeto} from '../projeto/projeto';

export class Portfolio {
  public id: number;
  public descricao: string;
  public criadoEm?: Date;
  public projetos: Projeto[] = [];

  constructor(
    id: number,
    descricao: string,
    criadoEm?: Date
  ) {
    this.id = id;
    this.descricao = descricao;
    this.criadoEm = criadoEm;
  }
}

File: back/src/PortfolioDev.Domain/Models/Portfolio.cs
using Microsoft.EntityFrameworkCore;
using PortfolioDev.Domain.Models.Identity;

namespace PortfolioDev.Domain.Models;

[Index(nameof(UsuarioId), IsUnique = true)]
public class Portfolio
{
	public int Id { get; set; }
	public string? Descricao { get; set; }
	public DateTime CriadoEm { get; set; }
	
	public int UsuarioId { get; set; }
	public Usuario Usuario { get; set; }
	
	public List<Projeto> Projetos { get; set; }
}